newdle/client/src/components/TopHeader.module.scss

```diff
@@ -8,3 +8,11 @@
     height: 2.5em;
   }
 }
+
+.logo {
+  content: url('../images/logo.svg');
+
+  @media (max-width: 768px) {
+    content: url('../images/logo_mobile.png');
```

Comment:
Why do we need a png for this? Shouldn't it be possible to just scale the SVG properly?

Reply:
Shouldn't this be solved by #144 anyway? Maybe this can just be removed.

Reply:
Indeed, please rebase to the latest master to have all the responsive fixes related to the front page and global elements.